src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/BatchConfiguration.kt

```diff
@@ -3,16 +3,30 @@ package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.config
 import org.springframework.batch.core.launch.JobLauncher
 import org.springframework.batch.core.launch.support.SimpleJobLauncher
 import org.springframework.batch.core.repository.JobRepository
+import org.springframework.batch.core.repository.support.JobRepositoryFactoryBean
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor
+import org.springframework.transaction.PlatformTransactionManager
+import javax.sql.DataSource
 
 @Configuration
 class BatchConfiguration(
+  private val dataSource: DataSource,
+  private val transactionManager: PlatformTransactionManager,
   @Value("\${spring.batch.concurrency.pool-size}") private val poolSize: Int,
   @Value("\${spring.batch.concurrency.queue-size}") private val queueSize: Int,
 ) {
+  @Throws(Exception::class)
+  protected fun createJobRepository(): JobRepository? {
```

Comment:
Hi @awhodgson256 , How does the datasource gets decided that it will be in memory?